nutanix/resource_nutanix_foundation_central_image_nodes.go

```diff
@@ -466,10 +826,35 @@ func resourceNutanixFCImageClusterCreate(ctx context.Context, d *schema.Resource
 	}
 	req.Timezone = utils.StringPtr(timezone.(string))
 
+	skipClusterCreation, ok := d.GetOk("skip_cluster_creation")
+	if !ok {
+		req.SkipClusterCreation = false
+	}
+	req.SkipClusterCreation = skipClusterCreation.(bool)
```

Comment:
This is overriding the changes done by if condition at line 830. By default req.SkipClusterCreation would be false when the struct is initialized.
```suggestion
	if skipClusterCreation, ok := d.GetOk("skip_cluster_creation"); ok {
	        req.SkipClusterCreation = skipClusterCreation.(bool)
	}
```

Reply:
suggestion for line 829-833